projects/aws-containers/hello-eks-anywhere/GIT_TAG

```diff
@@ -0,0 +1,2 @@
+v0.1.0
+
```

Comment:
Unnecessary spacing?

Reply:
We need newlines for the sedfiles, but I think this one is safe to remove.